content/doc/developer/plugin-governance/managing-permissions.adoc

```diff
@@ -0,0 +1,66 @@
+
+== GitHub Permissions
+
+GitHub permissions in Jenkins are managed by GitHub teams.
+Almost every plugin has a `${plugin-id} Developers` team created specifically for this plugin.
+This team may have a different link:https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization[permission level] depending on time of creation.
+For all new repositories we tend to grant `Admin` permissions at the moment, and other plugin teams can get permissions elevated upon a request.
+
+How to request GitHub permissions?
+You will need to send a permission transfer request and to specify the permission recipient's GitHub account and Jenkins LDAP(aka Jenkins Jira) account there.
+
+* Request permissions change in Jenkins Jira
+** Project: `INFRA`, Component: `github`
+* Request permissions in the Jenkins developer mailing list. 
+
+Permission change requests should be ideally done by one of the active plugin maintainers.
+Otherwise, an explicit approval will be required before the permission transfer happens.
+If there are multiple active maintainers in a plugin, Jenkins GitHub administrators will usually wait for a while to collect approvals from a number of them.
+
+=== Valid approvals by maintainers
+
+The following ways of approval are generally recognized as valid ones:
+
+* Confirmation in a Jenkins Jira ticket (permission request or other ticket)
+* Explicit approval in a GitHub pull request or issue
+** The reply should be sent from a GitHub account associated with a maintainer's Jenkins LDAP account
+* Confirmation in the developer mailing list.
+** The reply should be sent from the email listed in the Jenkins LDAP account
+
+Other types of approval (emails from different addresses, email forwards by requesters, etc.) will be reviewed and verified by `jenkinsci` GitHub administrators.
+
+== Release Permissions
+
+Permissions to upload plugin releases are independent of GitHub push access and maintained in the link:https://github.com/jenkins-infra/repository-permissions-updater[Repository Permissions Updater] repository.
+
+To request upload permissions for a new maintainer:
+
+. If you have never done it before, 
+  you log in at least in once with your Jenkins account into the link:https://repo.jenkins-ci.org/webapp/#/home[Jenkins artifact repository].
+  Any modification to the permission files will be ineffective until then.
+. File a PR on that repository in link:https://github.com/jenkins-infra/repository-permissions-updater[Repository Permissions Updater] as described in the README of that repository. Without this, you'll be unable to release your plugin.
+
+//TODO(oleg-nenashev): Add CoC violations and so on?
+
+By default we *DO NOT* remove upload permissions for current or past maintainers.
+It can be done based on the explicit approval of the previous maintainer or based on the discussion in the Developer mailing list.
```

Comment:
```suggestion
It can be done based on the explicit approval of the previous maintainer or based on a discussion in the link:https://groups.google.com/forum/#!forum/jenkinsci-dev[Developer mailing list].
```